#include "widget.h"
#include "ui_widget.h"
#include <QVBoxLayout>

Widget::Widget(QWidget *parent)
    : QWidget(parent)
    , ui(new Ui::Widget)
{
    ui->setupUi(this);

    // Create the Layout
    QVBoxLayout * layout = new QVBoxLayout();

    // Add Buttons to the Layout
    layout->addWidget(ui->oneButton);
    layout->addWidget(ui->twoButton);
    layout->addWidget(ui->threeButton);
    layout->addWidget(ui->fourButton);
    layout->addWidget(ui->fiveButton);

    // Set Layout to the Widget
    setLayout(layout);

}

Widget::~Widget()
{
    delete ui;
}

